server/server.py

```diff
@@ -69,6 +72,10 @@ class ServerServicer(core_pb2_grpc.CoreServicer):
 
     UNIMPLEMENTED_MSG = "Ah, ah, ah, you didn't say the magic word"
 
+    # fitted runtimes should be stored to disk, but that can't happen
+    # until we can guarantee that all primitives can be pickled.
+    fitted_runtimes: Dict[str, runtime.Runtime] = {}
```

Comment:
In the initial version of this, they were pickling the pipelines and saving them to the DB as a blob, using the fitted solution ID as the key.  I think the dict was introduced as a workaround when some of the primitives weren't pickling properly.  Is that an advisable course of action to follow once the pickling issues are worked out, or are we better off just writing to file?

Reply:
I think we do want to move away from the dictionary approach once we can pickle reliably. But I wouldn't want to conflate the two issues.